config/redirects.yml

```
@@ -348,6 +348,7 @@ redirects:
   "/blog/9-tips-to-help-you-apply-for-teacher-training": "/how-to-apply-for-teacher-training"
   "/blog/choosing-the-right-teacher-training-course-provider": "/train-to-be-a-teacher/how-to-choose-your-teacher-training-course"
   "/blog/how-to-find-school-experience": "/is-teaching-right-for-me/get-school-experience"
+  "/blog/what-to-expect-on-your-teacher-training": " /train-to-be-a-teacher/initial-teacher-training"
```

```suggestion
  "/blog/what-to-expect-on-your-teacher-training": "/train-to-be-a-teacher/initial-teacher-training"
```
@MaxineCl your redirect is currently throwing a 500 error - once you remove the extra space, it should clear up a build failure